client/src/store/new-request-module.ts

```diff
@@ -206,7 +206,7 @@ export class NewRequestModule extends VuexModule {
     'end_designation_more_than_once'
   ]
   displayedComponent: string = 'Tabs'
-  doNotAnalyzeEntities: string[] = ['PAR', 'CC', 'CP', 'PA', 'FI', 'XCP']
+  doNotAnalyzeEntities: string[] = ['PAR', 'CC', 'CP', 'PA', 'FI', 'XCP', 'SO']
```

Comment:
There may now be an enum for these.

Reply:
ah right! Will apply that.. here ;)

Reply:
Or not. That module is full of strings. It might be better to leave it self-consistent  🙄

Reply:
i definitely agree we should update it all.
Honestly, i don't mind doing this now. Are you ok if i do this, and delay this PR until either later today or tomorrow?

Reply:
I'd leave it to later but it's up to you.

Reply:
Okie doke, probably the wise play!